tool/cli.js

```diff
@@ -891,6 +893,61 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .argument("<output>", "Output to file or stdout", {
+    default: "stdout",
+  })
+  .action(
+    tryOrExit(async ({ args, options }) => {
+      const { verbose } = options;
+      const { output } = args;
+
+      if (verbose) {
+        console.log(chalk.green("Checking for precense of CONTENT_ROOT"));
+      }
+
+      if (!CONTENT_ROOT) {
+        throw new Error(
+          "CONTENT_ROOT not set. Please see the docs at tool/README.md"
+        );
+      }
+
+      if (verbose) {
+        console.log(chalk.green("CONTENT_ROOT set, generating inventory"));
+      }
+
+      const allPaths = klawSync(CONTENT_ROOT, {
+        nodir: true,
+        filter: (entry) => path.extname(entry.path) === ".md",
+        traverseAll: true,
+      });
+
+      const inventory = allPaths.map((entry) => {
+        const fileContents = fs.readFileSync(entry.path, "utf8");
+        const parsed = frontmatter(fileContents);
+
+        return {
+          path: entry.path.substring(entry.path.indexOf("/files")),
+          frontmatter: parsed.attributes,
+        };
+      });
+
+      if (output === "file") {
+        console.log(chalk.green("Writing ./frontmatter-invetory.json"));
+        fs.writeFileSync(
+          "./frontmatter-inventory.json",
+          JSON.stringify(inventory)
+        );
+        console.log(chalk.green("frontmatter-invetory.json written to disk."));
+      } else if (output === "stdout") {
+        process.stdout.write(JSON.stringify(inventory));
```

Comment:
Might be nice to make the output pretty?

```suggestion
        process.stdout.write(JSON.stringify(inventory, undefined, 2));
```